server/mvcc/metrics.go

```diff
@@ -52,6 +52,14 @@ var (
 			Help:      "Total number of puts seen by this member.",
 		})
 
+	putBigValue = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "etcd_debugging",
+			Subsystem: "mvcc",
+			Name:      "put_big_value",
+			Help:      "Every big value size of per key.",
+		},
+		[]string{"big_key"})
```

Comment:
It's high cardinality field. It has negative impact on performance: 

https://www.robustperception.io/cardinality-is-key#:~:text=Prometheus%20performance%20almost%20always%20comes,and%20POST%20in%20your%20application.

Exposing keys on a metric has also potential of leaking data (as they are exposed with auth).

Reply:
I make some code different, but I don't know how to deal cardinality field. Is it have some explicit identification.

Reply:
I don't think we should track any keys as a metric. 
It's still huge payload and still risk of unexpected security risk.

Reply:
Agreed, this would be high cardinality, what do you think of making this a counter of big values instead?  And you can log the big_key value instead of putting it as metric value, so if you really want to alert on this to debug you can look at the logs.